### PR TITLE
epic: cumulative update 2.0  

### PR DESCRIPTION
**Pull request by ala-n**

### CODE REVIEW

**Review by AlexanderBazukevich — APPROVED** [review]

File: src/core/panel/plugin-panel.header.less [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    left: 0;
+    width: 100%;
+    height: 100%;
+    z-index: 1;
+
+    background: transparent;
+    border: none;
+    box-shadow: none;
+    appearance: none;
+  }
```
Comment (yadamskaya) [inline src/core/panel/plugin-panel.header.less:53]: Can we add `cursor: pointer;` here?

File: src/plugins/editor/editor.tsx [hunk 2]
```diff
@@ -66,15 +101,23 @@ export class UIPEditor extends UIPPlugin {
...
+  /** Callback to call on an editor's content changes */
   @decorate(debounce, 1000)
   protected _onChange(): void {
     this.model!.setHtml(this.value, this);
   }
 
   /** Change editor's markup from markup state changes */
-  @bind
-  protected override _onRootStateChange(): void {
+  @listen({event: 'uip:change', target: ($this: UIPEditor)=> $this.$root})
```
Comment (yadamskaya) [inline src/plugins/editor/editor.tsx:119]:
```suggestion
  @listen({event: 'uip:change', target: ($this: UIPEditor) => $this.$root})
```

File: src/plugins/settings/select-setting/select-setting.less [hunk 3]
```diff
@@ -9,36 +7,29 @@
```
[the remark this answers is not in the document]
Comment (ala-n) [reply src/plugins/settings/select-setting/select-setting.less:7]: This needs to be addressed to @exadel/esl project
TODO: @yadamskaya or @exadel-inc/esl-core-maintainers to create issue

**Comment by ala-n** [comment]
PR cumulative review moved under main-beta -> main request

**Pull request merged**